go/src/koding/kites/config/cache.go

```diff
@@ -83,7 +83,7 @@ func newBoltDB(o *CacheOptions) (*bolt.DB, error) {
 	return bolt.Open(o.File, 0644, o.BoltDB)
 }
 
-func KodingHome() string {
+func KodingConfigHome() string {
```

Comment:
I'm just nitpicking about the naming, as always. Having cache under ~/.cache is cool, nothing to do here. But there's no need to rename KodingHome because of it, that's was my point.